### PR TITLE
HEMA Master profession

### PR DESCRIPTION
**Pull request by ghost**
#### Summary

SUMMARY: Content "Add HEMA Master profession to the game"
#### Purpose of change

I thought this would be an interesting profession for some people, so I added it.
#### Describe the solution

Make a HEMA Master profession with medieval armor and sword skills.
#### Describe alternatives you've considered

No medieval knight LARPing?
#### Testing

Spawned in game as profession, worked.

### CODE REVIEW

File: data/json/professions.json [hunk 1]
```diff
@@ -4147,5 +4147,21 @@
       "female": [ "sports_bra", "boy_shorts" ]
     },
     "flags": [ "SCEN_ONLY" ]
+  },
+  {
+    "type": "profession",
+    "ident": "knight",
+    "name": "Medieval Knight",
+    "description": "You've made medieval swordsmanship your life. Now you will finally have a real use for it.",
```
Comment (anothersimulacrum) [inline data/json/professions.json:4155]:
```suggestion
    "description": "You've made medieval swordsmanship your life.  Now you will finally have a real use for it.",
```

Comment (Talvan) [reply data/json/professions.json:4155]: I'm sorry if I'm misunderstanding something here but why does it need an extra space there?

Comment (anothersimulacrum) [reply data/json/professions.json:4155]: It's a style standard we have - two spaces after a period. It has something to do with monospaced fonts.

Comment (Talvan) [reply data/json/professions.json:4155]: Alright, thanks.

**Comment by waveyl** [comment]
Is 8 points too many?  It's equal to bionic assassin.  But bionic assassin do better in combat, and he has many useful bionics.

**Comment by Shodan14** [comment]
Broadsword is a bit later than medieval, I'd recommend a longsword.

**Comment by LaVeyanFiend** [comment]
Wouldn't this be better titled as HEMA Practitioner/Artist/Master? Less implied time travel shenanigans that way :P

**Comment by Talvan** [comment]
I'll go with HEMA Master and give it a longsword instead of a broadsword.

**Comment by Testudo-Cultu** [comment]
> Wouldn't this be better titled as HEMA Practitioner/Artist/Master? Less implied time travel shenanigans

How do you justify the medieval peasant starting scenario then?

**Comment by kevingranade** [comment]
It's a fair question that applies to the pesant as well.  It's terribly inconsistent.

**Comment by Talvan** [comment]
> Is 8 points too many? It's equal to bionic assassin. But bionic assassin do better in combat, and he has many useful bionics.

What would you suggest as a better point cost? 7 or 6?

**Comment by waveyl** [comment]
> > Is 8 points too many? It's equal to bionic assassin. But bionic assassin do better in combat, and he has many useful bionics.
> 
> What would you suggest as a better point cost? 7 or 6?

6, I think, 2 point for an additional trait.

**Comment by Dacendeth** [comment]
8 seems fine to me, you start with two really good layers of armor that unevolved zeds probably wont be able to even scratch you, a great weapon and the skills to use it with.  I'd argue that the knight is on par with the assassin because of this and the 8 cost is fair.

**Pull request closed without merging**